README.md

```diff
@@ -35,231 +31,13 @@ Stable releases are also tagged on this repository:
-clang -g -shared -framework Cocoa -Wl,-undefined,dynamic_lookup src/test.os -o lib/test.dylib
-```
-
-- `-g` is for debugging information.
-- Use `godot_nativescript_*` methods only in the `nativescript_init()` function.
-
-### Create GDNativeLibrary Resource
-The GDNativeLibrary resource contains links to the libraries for each platform.
-
-1. Create a new resource in memory and edit it.
-2. Select `Resource > GDNativeLibrary`.
-3. Set the library file for your platform inside the inspector.
-4. Save the edited resource as a `.tres`
-
-<details>
-
-![](images/faq/dllibrary_create_new_resource.png?raw=true)
-
-![](images/faq/dllibrary_create_new_dllibrary.png?raw=true)
-
-![](images/faq/dllibrary_save_as_resource.png?raw=true)
-
-*Note*: Remember to save `GDNativeLibrary` as `.gdnlib`
-
-</details>
-
-Expand *Details* for screenshots.
-
-### Using GDNativeLibrary in GDScript
-
-```gdscript
-extends Node
-
-func _ready():
-	var gdn = GDNative.new()
-	gdn.library = load("res://lib/libtest.tres")
-
-	gdn.initialize()
-
-	var res = gdn.call_native("standard_varcall", "some_test_procedure", [])
-
-	print("result: ", res)
-
-	gdn.terminate()
-```
-
-### Attaching GDNativeLibrary to a Node
-
-1. Attach a new script to a node.
-2. In the pop-up dialog, choose NativeScript in the `Language` menu.
-3. Enable built-in script, or create a `.gdn` file, which only contains a name.
-4. Specify the `Class Name`.
-5. Press `Create`.
-
-The GDNativeLibrary field in a NativeScript is empty by default.
-
-
-<details>
-
-![](images/faq/create_dlscript.png?raw=true)
-
-![](images/faq/set_script_dllibrary.png?raw=true)
-
-</details>
-
-Expand *Details* for screenshots.
-
-## FAQ
-
-**What is the difference between `GDNative` and `NativeScript`?**
-
-`GDNative` is a new class that can call native functions in libraries.
-GDScript / VisualScript / C#, etc, are able to use this class.
-
-Godot treats `NativeScript` as a scripting language, enabling the
-use of GDNative to implement scripts backed by native code.
-
-**Which languages are binding as a NativeScript?**
-
-[**C++**](https://github.com/godotengine/godot-cpp),
-[**D**](https://github.com/godot-d/godot-d),
-[**Nim**](https://github.com/pragmagic/godot-nim)
-
-**Can you debug NativeScripts?**
-
-You must compile the library with debug
-symbols, and then you can use your debugger as usual.
-
-**Can you use one GDNativeLibrary for all NativeScripts?**
+## Using this library
 
-You can! ✨
+This library is meant to be used in conjunction with a code generator that creates the binding code necesairy to interact with Godot. This library only provides access to the API over which communication with the Godot core is implemented and binding meta data in the form of a json document.
 
-**What is the reason behind the name "GDNative"?**
+One such binding library is [godot-cpp](https://github.com/godotengine/godot/pull/47455) which implements the bindings using C++.
```

Comment:
Wrong URL.

Reply:
wtf, I swear my copy/paste buffer is so unreliable these days... I sometimes have to press CTRL-C a few times before it decides to overwrite what is currently in my copy buffer.